Web/Pages/Transactions/TransactionForm.razor

```diff
@@ -222,6 +222,10 @@
                    Variant="Variant.Filled">
             Upload new attachment(s)
         </MudButton>
+        @if (FileErrorMessage != null)
+        {   
+            <p class="errormessage">@FileErrorMessage</p>
```

Comment:
We should use MudBlazor's theme colors here: https://mudblazor.com/features/colors

Reply:
Also in the case of plain old css?